crates/model/src/orders/stop_limit.rs

```diff
@@ -509,3 +589,126 @@ impl Display for StopLimitOrder {
         )
     }
 }
+
+////////////////////////////////////////////////////////////////////////////////
+//  Tests
+////////////////////////////////////////////////////////////////////////////////
+#[cfg(test)]
+mod tests {
+    use rstest::rstest;
+
+    use crate::{
+        enums::{OrderSide, OrderType, TimeInForce, TriggerType},
+        instruments::{CurrencyPair, stubs::*},
+        orders::{Order, builder::OrderTestBuilder},
+        types::{Price, Quantity},
+    };
+
+    #[rstest]
+    fn buy_breakout_ok(_audusd_sim: CurrencyPair) {
```

Comment:
@cjdsellers let me know what do you think about this type of test, should I leave, remove or add more like this?

Reply:
the reason I am asking is because I haven't seen smth like this here

https://github.com/nautechsystems/nautilus_trader/blob/develop/crates/model/src/orders/market.rs#L539